friendly/core.py

```diff
@@ -640,7 +640,7 @@ def assign_tracebacks(self):
             for line in python_tb:  # excluding our own code
                 if exclude and line.strip() == "exec(code, self.locals)":
                     continue
-                exclude = any(filename in line for filename in EXCLUDED_FILE_PATH)
+                exclude = any(str(filename) in line for filename in EXCLUDED_FILE_PATH)
```

Comment:
This needs to be modified as it does not consider files excluded from an entire directory. I also realize that the original version had a bug that might exclude a line of code such as:
```python
print("some/excluded/path")
```

I will need to write some unit tests to cover this.

Rather than doing a conversion to string here as well as a conversion to a pathlib object when storing the paths to be excluded, I prefer to do a single conversion to string when storing the paths, keeping them as string everywhere.

Reply:
I see and understand, then I'll leave this here be for now :)